etc/config/c++.amazon.properties

```diff
@@ -336,272 +339,188 @@ compiler.clang_embed.semver=(std::embed)
+# An alias of this name needs to be maintained to allow old URLs to continue to work
 compiler.armv8-clang900.alias=armv8-clang-9
-
-compiler.armv7-clang-trunk.name=armv7-a clang (trunk)
-compiler.armv7-clang-trunk.exe=/opt/compiler-explorer/clang-trunk/bin/clang++
-compiler.armv7-clang-trunk.demangler=/opt/compiler-explorer/gcc-snapshot/bin/c++filt
-compiler.armv7-clang-trunk.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
-compiler.armv7-clang-trunk.semver=(trunk)
-# Arm v7-a with Neon and VFPv3
-compiler.armv7-clang-trunk.options=-target arm-linux-gnueabi --gcc-toolchain=/opt/compiler-explorer/arm/gcc-8.2.0/arm-unknown-linux-gnueabi --sysroot=/opt/compiler-explorer/arm/gcc-8.2.0/arm-unknown-linux-gnueabi/arm-unknown-linux-gnueabi/sysroot
-
-compiler.armv8-clang-trunk.name=armv8-a clang (trunk)
 compiler.armv8-clang-trunk.exe=/opt/compiler-explorer/clang-trunk/bin/clang++
 compiler.armv8-clang-trunk.demangler=/opt/compiler-explorer/gcc-snapshot/bin/c++filt
 compiler.armv8-clang-trunk.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
 compiler.armv8-clang-trunk.semver=(trunk)
-# Arm v8-a
 compiler.armv8-clang-trunk.options=-target aarch64-linux-gnu --gcc-toolchain=/opt/compiler-explorer/arm64/gcc-8.2.0/aarch64-unknown-linux-gnu --sysroot=/opt/compiler-explorer/arm64/gcc-8.2.0/aarch64-unknown-linux-gnu/aarch64-unknown-linux-gnu/sysroot
-
-compiler.armv8-full-clang-trunk.name=armv8-a clang (trunk, all architectural features)
 compiler.armv8-full-clang-trunk.exe=/opt/compiler-explorer/clang-trunk/bin/clang++
 compiler.armv8-full-clang-trunk.demangler=/opt/compiler-explorer/gcc-snapshot/bin/c++filt
 compiler.armv8-full-clang-trunk.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
-compiler.armv8-full-clang-trunk.semver=(trunk allfeats)
+compiler.armv8-full-clang-trunk.semver=(trunk, all architectural features)
 # Arm v8-a with all supported architectural features
 compiler.armv8-full-clang-trunk.options=-target aarch64-linux-gnu --gcc-toolchain=/opt/compiler-explorer/arm64/gcc-8.2.0/aarch64-unknown-linux-gnu --sysroot=/opt/compiler-explorer/arm64/gcc-8.2.0/aarch64-unknown-linux-gnu/aarch64-unknown-linux-gnu/sysroot -march=armv8.6-a+crypto+rcpc+sha3+sm4+profile+rng+memtag+sve2+sve2-bitperm+sve2-sm4+sve2-aes+sve2-sha3+tme
-# An alias of the original name for this compiler needs to be maintained to
-# allow old URLs to continue to work
+# An alias of the original name for this compiler needs to be maintained to allow old URLs to continue to work
 compiler.armv8-full-clang-trunk.alias=armv8.5-clang-trunk
 
+################################
 # Clang for RISC-V
-group.rvclang.compilers=rv32-clang:rv32-clang1101:rv32-clang1100:rv32-clang1001:rv32-clang1000:rv32-clang901:rv32-clang900:rv64-clang:rv64-clang1101:rv64-clang1100:rv64-clang1001:rv64-clang1000:rv64-clang901:rv64-clang900
+group.rvclang.compilers=&rv32clang:&rv32clang
 group.rvclang.groupName=RISC-V Clang
```

Comment:
I'm not 100% sure that this works for subgroups. Do both of them end up in the same optgroup in the compiler selector?

Reply:
is this also not meant to say rv64clang on the second one?

That said - I don't think there's much support for subgroups in general. So I would be hesitant to try it out

Reply:
And this is exactly why it's a PR, I suck at spotting my sloppy c&p. Thank you!

Reply:
![imagen](https://user-images.githubusercontent.com/5364255/124383899-29a5ba00-dccf-11eb-92ef-e449f202e64a.png)
Argh